tests/publiccloud/check_registercloudguest.pm

```diff
@@ -76,7 +76,7 @@ sub run {
             $instance->ssh_assert_script_run(cmd => '! sudo SUSEConnect -d', fail_message => 'SUSEConnect succeeds but it is not supported should fail on BYOS');
         }
     } else {
-        if ($instance->ssh_script_output(cmd => 'sudo zypper lr | wc -l', timeout => 360) < 5) {
+        if ($instance->ssh_script_output(cmd => 'sudo zypper -x lr | grep "<repo" | wc -l', timeout => 360) < 3) {
```

Comment:
TBH, I do not understand why we want to ensure that there are only three lines of repositories displayed. This not being the case does not appear as a valid criterion for a test to fail. It does not appear a breaking criterion for an OS to have a limited number of repos available...

Is it possible to check for anything that is more substantial?

Reply:
Sorry for the misunderstanding from my side, indeed  it checks if there are at least 3 repos and not at max.

We can handle the clarification in another ticket.